test/core/networking/protocol/authentication_test.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:test/test.dart';
+
+import 'package:flyde/core/networking/protocol/authentication.dart';
+import 'package:flyde/core/networking/protocol/protocol.dart';
+
+void main() {
+  group('AuthRequest', () {
+    test('In ProtocolDelegate.elements', () {
```

Comment:
Description should be a full sentence describing what is tested. There's missing what is in elements. First part is for all test descriptions.

Reply:
Not really, this will print: 'AuthRequest in ProtocolDelegate.elements' to the console if this fails, which is pretty accurate. (The context is given with the group)

Reply:
'AuthRequest in ProtocolDelegate.elements' still not a sentence